parlai/crowdsourcing/tasks/turn_annotations/impl.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
```

Comment:
hmm - what's the advantage of putting this in a different script than `run.py`? (And curious - what does `impl` stand for? :) )

Reply:
Implementation. This used to be `util` but that name didn't make sense here. Only have it in a separate script because that's how it was like in the original.